test/suites/network/multi-nic.robot

```diff
@@ -31,15 +34,57 @@ Verify MicroShift Runs On Both NICs
     # Wait for MicroShift API readiness and run verification
     Wait For MicroShift
     Verify Hello MicroShift LB
-    Verify Hello MicroShift NodePort    ${USHIFT_HOST_IP1}    ${USHIFT_HOST_IP2}
+    Verify Hello MicroShift NodePort    ${USHIFT_HOST_IP1}
+    Verify Hello MicroShift NodePort    ${USHIFT_HOST_IP2}
+    Check IP Present In External Certificate    ${USHIFT_HOST_IP1}
+    Check IP Present In External Certificate    ${USHIFT_HOST_IP2}
 
 Verify MicroShift Runs On First NIC
     [Documentation]    Verify MicroShift can run on the first NIC
-    Verify MicroShift On One NIC    ${USHIFT_HOST_IP1}    ${NIC2_NAME}    ${USHIFT_HOST_IP1}    ${EMPTY}
+    [Setup]    Save Default MicroShift Config
+
+    Setup MicroShift Config    ${USHIFT_HOST_IP1}
+
+    ${cur_pid}=    MicroShift Process ID
+
+    Login Switch To IP    ${USHIFT_HOST_IP1}
+    Nmcli Connection Control    down    ${NIC2_NAME}
+
+    Restart MicroShift
+
+    Verify MicroShift On One NIC    ${USHIFT_HOST_IP1}    ${USHIFT_HOST_IP2}
+
+    # Rebooting MicroShift host restores the network configuration
+    [Teardown]    Run Keywords
+    ...    Restore Default MicroShift Config
+    ...    Reboot MicroShift Host
+    ...    Login Switch To IP1
+    ...    Wait For Healthy System
 
 Verify MicroShift Runs On Second NIC
```

Comment:
I'm wondering what is functional difference between two tests so we could name them better (or improve documentation).

They have different flows, but it's not obvious what exactly we're testing: why in one test we need to explicitly restart microshift, but in another it restarts itself.
It would be nice that this difference would be named other than "First NIC" and "Second NIC"

Reply:
Added some documentation comments and swapped first/second with primary/secondary.